README.md

```diff
@@ -99,94 +89,98 @@ The tests directory (augur-core/tests) contain tests and test fixtures to test t
 - test_controller.py -- tests controller functionalities.
 - test_mutex.py -- tests mutex functionalities.
 - test_helpers.py -- tests the controller, safeMath, and assertNoValue macros.
-- test_legacyRep.py -- tests the legacyRepTokens’ functionalities.
+- test_legacyRep.py -- tests the legacyRepTokens' functionalities.
 - utils.py -- contains useful functions for testing, such as conversion between different data types.
 - wcl-in-python.py -- contains functions for making and taking various types of bids.
 - wcl.txt -- explains tests for the various situations when filling a bid and filling an ask.
 
-In order to run Augur’s test suite, run the following inside the tests directory:
-```
+In order to run Augur's test suite, run the following inside the tests directory:
+
+```bash
 cd augur-core/tests
 pytest
 ```
+
 This executes all the tests. To run a test individually, run the following:
-```
-pytest path/to/test_file.py -k 'name_of_test’
-```
 
-By default the trading WCL fuzz testing is not run, if you want to run it specifically do
+```bash
+pytest path/to/test_file.py -k 'name_of_test'
 ```
-export INCLUDE_FUZZY_TESTS=1
-pytest trading/test_wcl_fuzzy.py
-unset INCLUDE_FUZZY_TESTS
+
+The unit test suite includes tests that use a fuzzer, which take somewhat longer (around 10 minutes on CI) to run than the rest of the tests.  These tests are _not_ run by default; to include these tests, set the environment variable `INCLUDE_FUZZY_TESTS=1`:
```

Comment:
Again not you, but `INCLUDE_FUZZY_TESTS` is no longer necessary as the flag was removed.